### ISSUE TITLE
[bug] Conan 2.2.0: ModuleNotFoundError: No module named 'distutils'

### ISSUE THREAD

**Issue by jwillikers** [conan-io/conan#15905]
### Describe the bug

Environment: Fedora Linux 39, Python 3.12, Conan 2.2

```
 conan --version
Traceback (most recent call last):
  File "/home/jordan/conan2-venv/bin/conan", line 5, in <module>
    from conans.conan import run
  File "/home/jordan/conan2-venv/lib64/python3.12/site-packages/conans/conan.py", line 3, in <module>
    from conan.cli.cli import main
  File "/home/jordan/conan2-venv/lib64/python3.12/site-packages/conan/cli/cli.py", line 13, in <module>
    from conan.api.conan_api import ConanAPI
  File "/home/jordan/conan2-venv/lib64/python3.12/site-packages/conan/api/conan_api.py", line 3, in <module>
    from conan.api.subapi.cache import CacheAPI
  File "/home/jordan/conan2-venv/lib64/python3.12/site-packages/conan/api/subapi/cache.py", line 10, in <module>
    from conan.internal.conan_app import ConanApp
  File "/home/jordan/conan2-venv/lib64/python3.12/site-packages/conan/internal/conan_app.py", line 11, in <module>
    from conans.client.remote_manager import RemoteManager
  File "/home/jordan/conan2-venv/lib64/python3.12/site-packages/conans/client/remote_manager.py", line 8, in <module>
    from conans.client.rest_client_local_recipe_index import RestApiClientLocalRecipesIndex
  File "/home/jordan/conan2-venv/lib64/python3.12/site-packages/conans/client/rest_client_local_recipe_index.py", line 4, in <module>
    from distutils.dir_util import copy_tree
ModuleNotFoundError: No module named 'distutils'
```

Introduced in #13930.

### How to reproduce it

Install Conan 2.2.0 using Python 3.12 and run `conan --version`.

**Comment by czoido** [comment]
Hi @jwillikers,
Thanks a lot for reporting, we'll try to fix this ASAP.

**Comment by czoido** [comment]
Closed by https://github.com/conan-io/conan/pull/15906 will be released in a couple of minutes with Conan 2.2.1
